### ISSUE TITLE
Ability to alias a list of commands

### ISSUE THREAD

**Issue by szaydel** [boothj5/profanity#381]
One thing that would really be helpful I think to me and I am sure others is ability to alias commands, much like shell allows you to do. Custom aliases would I think help with personalizing a lot of things and help out with workflow. Even more helpful would be ability to create a custom command that strings together a few commands to create a small process.

**Comment by boothj5** [comment]
This hasn't been implemented yet since it is probably more in the territory of building custom plugins.

Support for plugins is still in development: https://github.com/boothj5/profanity-plugins

**Comment by longk15t** [comment]
How to set multiple commands into an alias?  Above link just says one command as one alias.

**Comment by boothj5** [comment]
I've started adding code to enable this feature using scripts with lists of commands as per #556 

Scripts should contain one command per line, and need to be saved in `$XDG_DATA_HOME/profanity/scripts/` which is usually `$HOME/.local/share/profanity/scripts/` on most systems.

To execute a script:

```
/script run <script>
```

There's still work to do, enable autocomplete, and list/remove/show etc, see:

```
/help script
```

I've also added an account property `startscript` which allows you to set a script to run every time an account logs in. I've found this useful for autojoining rooms on XMPP gateways that do not support bookmarks for example.  To assign a start script to an account:

```
/account set <account> startscript <script>
```

I'll keep this issue up to date, any feedback welcome.
